ddp-workspace/projects/ddp-dsm-ui/src/app/file-download/file-download.component.ts

```diff
@@ -39,8 +39,8 @@ export class FileDownloadComponent {
 
     const realm = localStorage.getItem( ComponentService.MENU_SELECTED_REALM );
     this.dsmService.getSignedUrl( this.participant.data.profile[ 'guid' ], file, realm ).pipe(
-      mergeMap(data => this.dsmService.downloadFromSignedUrl(data['url']).pipe(take(1)))).subscribe( {
-        next: data => {
+      mergeMap(data => this.dsmService.downloadFromSignedUrl(data['url']), take(1))).subscribe( {
```

Comment:
Incorrect fix here.

It should be like:
![image](https://user-images.githubusercontent.com/7396837/184208026-5cd4faf4-3259-4d45-a09f-cebbe7168cd9.png)